### ISSUE TITLE
🐞 Bug version.rb cannot find ruby_libversion

### ISSUE THREAD

**Issue by uberhacker** [chromebrew/chromebrew#9941]
```
$ ./version.rb 
<internal:/usr/local/lib64/ruby/3.3.0/rubygems/core_ext/kernel_require.rb>:136:in `require': cannot load such file -- ruby_libversion (LoadError)
Did you mean?  rubygems/version
	from <internal:/usr/local/lib64/ruby/3.3.0/rubygems/core_ext/kernel_require.rb>:136:in `require'
	from ./version.rb:12:in `<main>'
```
Related?
```
$ gem install ruby-libversion
Building native extensions. This could take a while...
ERROR:  Error installing ruby-libversion:
	ERROR: Failed to build gem native extension.

    current directory: /usr/local/lib64/ruby/gems/3.3.0/gems/ruby-libversion-1.0.0/ext/libversion
/usr/local/bin/ruby extconf.rb
checking for pkg-config for libversion... not found
creating Makefile

current directory: /usr/local/lib64/ruby/gems/3.3.0/gems/ruby-libversion-1.0.0/ext/libversion
make DESTDIR\= sitearchdir\=./.gem.20240610-858-dp0xke sitelibdir\=./.gem.20240610-858-dp0xke clean

current directory: /usr/local/lib64/ruby/gems/3.3.0/gems/ruby-libversion-1.0.0/ext/libversion
make DESTDIR\= sitearchdir\=./.gem.20240610-858-dp0xke sitelibdir\=./.gem.20240610-858-dp0xke
compiling libversion.c
libversion.c:2:10: fatal error: libversion/version.h: No such file or directory
    2 | #include "libversion/version.h"
      |          ^~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
make: *** [Makefile:248: libversion.o] Error 1

make failed, exit code 2

Gem files will remain installed in /usr/local/lib64/ruby/gems/3.3.0/gems/ruby-libversion-1.0.0 for inspection.
Results logged to /usr/local/lib64/ruby/gems/3.3.0/extensions/x86_64-linux-gnu/3.3.0/ruby-libversion-1.0.0/gem_make.out
```

**Comment by satmandu** [comment]
Where is `libversion/version.h` ?

**Comment by Zopolis4** [comment]
Oh, it needs https://github.com/repology/libversion. Forgot to package that.

**Comment by Zopolis4** [comment]
(In the middle of fixing my build workflow, will get around to it when I can)

**Comment by uberhacker** [comment]
This is still an issue with the latest crew updates.
```
$ crew check -V mesa
<internal:/usr/local/lib64/ruby/site_ruby/3.3.0/rubygems/core_ext/kernel_require.rb>:136:in `require': cannot load such file -- ruby_libversion (LoadError)
Did you mean?  rubygems/version
	from <internal:/usr/local/lib64/ruby/site_ruby/3.3.0/rubygems/core_ext/kernel_require.rb>:136:in `require'
	from ../tools/version.rb:12:in `<main>'
```

**Comment by uberhacker** [comment]
FWIW, the ruby gem works.
```
$ gem install ruby-libversion
Fetching ruby-libversion-1.0.0.gem
Building native extensions. This could take a while...
Successfully installed ruby-libversion-1.0.0
1 gem installed
$ crew check -V mesa
Package                            Status              Current             Upstream
-------                            ------              -------             --------
mesa                               outdated            24.1.1-llvm18       24.1.1
```

**Comment by Zopolis4** [comment]
I'm not sure what the issue is here-- `version.rb` depends on `ruby-libversion`, which depends on `libversion`. 

Chromebrew provides libversion, and rubygems provides ruby-libversion.

`crew install libversion && gem install ruby-libversion` will install the dependencies.

**Comment by Zopolis4** [comment]
Should be fixed by #10447.
